### ISSUE TITLE
frontend change with Login/Signup

### ISSUE THREAD

**Issue by shang8024** [dcsil/B2ST-App#42]
Here are the current problems with our login/sigup frontend, violating 3.3.1 and 3.3.2 in WCAG 2.0 checklist Level A 

- [x] Lack input validation before sending request to server.

- [x] The way error messages pop up is not very decent. Using MUI alert, or setting error prop in MUI input element will help.

- [x] Hint text. It's hard for user to know how "complex" a password should be.

- [x] Lack check for the checkbox

- [x] Bad logout UI. Shoudl add to DashboardAppBar.

Check props of MUI input element, especially error, validator, and hintext.

Due is before Wed 11.30 11:59pm. But should complete subtasks stage by stage.
